designsafe/static/scripts/data-depot/components/projects/pipeline-publish/pipeline-publish.component.js

```diff
@@ -42,52 +42,54 @@ class PipelinePublishCtrl {
             license: this.resolve.license,
         };
 
-        let uuids = Object.keys(this.selectedListings);
-        uuids.forEach((uuid) => {
-            let listing = this.selectedListings[uuid];
-            let entity = this.project.getRelatedByUuid(uuid);
-            let attr = attributeMap[entity.name];
-            let pubEntity = { name: entity.name, uuid: entity.uuid };
-            pubEntity.fileObjs = _.map(listing.children, (child) => {
-                return {
-                    name: child.name,
-                    path: child.path,
-                    system: child.system,
-                    type: child.type
-                };
+        if (this.project.value.projectType !== 'other') {
```

Comment:
Added a conditional here for Other projects since there is a different format for `selectedListings`.